Sources/Web3Core/KeystoreManager/EtherscanTransactionChecker.swift

```diff
@@ -12,17 +12,17 @@ public struct EtherscanTransactionChecker: TransactionChecker {
 
     public init(urlSession: URLSession, apiKey: String) {
         self.urlSession = URLSessionProxyImplementation(urlSession: urlSession)
-        self.apiKey = apiKey
+        self.apiKey = apiKey.trim()
     }
 
     internal init(urlSession: URLSessionProxy, apiKey: String) {
         self.urlSession = urlSession
-        self.apiKey = apiKey
+        self.apiKey = apiKey.trim()
```

Comment:
I'd purpose to switch all such string functions as a properties. e.g. `apiKey.trimed`, `function.selector.hexPrefixAdded.lowercased`. I mean to add such as a separate issue for further work.

Reply:
I do not mind but what is the benefit or reason for it?

Reply:
This way it would better fits with clean code rules. Actually this way it wouldn't violates the rule of in-depth abstraction layers mixing as it is with chain function calls.

Reply:
To be handled in a separate PR.